js/mojs-player.babel.js

```diff
@@ -244,7 +248,14 @@ class MojsPlayer extends Module {
     @private
     @param {Object} Original event object.
   */
-  _onSeekStart ( e ) { this._sysTween.pause(); }
+  _onSeekStart ( e ) {
+    this._sysTween.pause();
+
+    const { onSeekStart } = this._props;
+    if (onSeekStart) {
```

Comment:
Let's be sure an actual `function type` was passed as callback here, we can have `_isFunction` helper and use it for all 4 examples like this:

```javascript
  const { onSeekStart } = this._props;
  if (this._isFunction(onSeekStart)) {
    // run
  }
```

The `_isFunction` can be as simple as:
```javascript
  // ...
  _isFunction (testVariable) {
    return typeof testVariable === 'function';
  }
  // ...
```